chrome/background.js

```diff
@@ -0,0 +1,46 @@
+/*
+Background.html
```

Comment:
so... this doesn't seem like an html page, and it has `.js` as an extension... ?